src/main/java/nebula/plugin/metrics/MetricsPlugin.java

```diff
@@ -58,9 +64,16 @@ public MetricsDispatcher get() {
         }
     };
 
+    @Inject
+    BuildScopeServices buildScopeServices;
```

Comment:
@rpalcolea What is this used for? It would be good if this plugin didn't depend on any Gradle internals.